<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorTableViewWidget</class>
 <widget class="QWidget" name="MonitorTableViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="findEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="findButton">
         <property name="text">
          <string>Find</string>
         </property>
         <property name="icon">
          <iconset resource="pics.qrc">
           <normaloff>:/Icons/Actions/Find</normaloff>:/Icons/Actions/Find</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="monitorTableView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
         <property name="icon">
          <iconset resource="pics.qrc">
           <normaloff>:/Icons/Actions/Add</normaloff>:/Icons/Actions/Add</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="icon">
          <iconset resource="pics.qrc">
           <normaloff>:/Icons/Actions/Remove</normaloff>:/Icons/Actions/Remove</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
